crucible-llvm/src/Lang/Crucible/LLVM/Translation/Instruction.hs

```diff
@@ -1601,6 +1603,12 @@ generateInstr retType lab defSet instr assign_f k =
       do callFunction defSet instr False fnTy fn args assign_f
          definePhiBlock lab normLabel
 
+    L.CallBr fnTy fn args normLabel otherLabels -> do
+      do callFunction defSet instr False fnTy fn args assign_f
+         _ <- definePhiBlock lab normLabel
+         for_ otherLabels $ \lab' -> void (definePhiBlock lab lab')
+         k
```

Comment:
Given that `CallBr` is a block-terminating instruction, I don't think we should invoke `k` here. `Invoke`'s case doesn't invoke `k` either, which makes me especially suspicious.

Reply:
Good catch, thanks!